app/javascript/packages/unpolyfill-webpack-plugin/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@18f/identity-unpolyfill-webpack-plugin",
```

Comment:
> new package: add a README for it?

Good idea!

(Also makes me wonder if we should lint for this)

Reply:
I'm in favor of that! I thought I had asked that in the past, but when I found [the comment](https://github.com/18F/identity-idp/pull/6701#discussion_r940387341), it was only on `package.json` contents, not the directory contents.

Reply:
Yeah, it was probably left out just because so few of the packages had README's at the time that it'd take a fair bit of work. We've added several since, so hopefully it wouldn't be a huge lift to flesh out the remaining. At worst, we could create an exclude list while still enforcing it for new packages.

Reply:
Added documentation in 7949f01, and enforcement of documentation for new packages in 6186a42.